ovos_document_chunkers/text/paragraphs.py

```
@@ -28,18 +28,21 @@ def __init__(self, config: Optional[Dict] = None):
+            An iterable of paragraph strings.
         """
         for c in data.split("\n\n"):
-            for p in paragraph_tokenize(c):
-                yield p
-
+            if not c.strip():
+                continue
+            try:
+                for p in paragraph_tokenize(c):
+                    yield p
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)